### DISCUSSION TITLE
Adding the "Pause" subscription option to customer account

### DISCUSSION THREAD

**Discussion by flonUK** [swellstores/community#486] [Help] [answered]
Hi all,
Could anyone please suggest how to add the pause subscription option to the customer account dashboard for the Horizon theme? I believe this was removed as part of the Swell hosting.
<img width="1103" alt="Screenshot 2024-01-30 at 09 14 46" src="https://github.com/swellstores/community/assets/156616226/99e3766e-2c83-466f-914c-ec4446d5a943">

Customers currently only have the option to cancel.

Thank you

**Comment by logeshswell** [comment, marked as answer]
It is a gap in our implementation. You can add support for it by following the below doc in GraphQL.
https://developers.swell.is/frontend-api/subscriptions#pause-a-subscription
